plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/ServerSettingsWizardPage.java

```diff
@@ -497,9 +495,9 @@ private SelectionListener onImportProject(ServerSettingsWizardPageModel model, f
             @Override
             public void widgetSelected(SelectionEvent e) {
                 Map<com.openshift.restclient.model.IProject, Collection<IBuildConfig>> projectsAndBuildConfigs = new HashMap<>();
-                List<IBuildConfig> buildConfigs = new ArrayList<>();
-                ResourceUtils.getBuildConfigFor(model.getResource(), buildConfigs);
-                projectsAndBuildConfigs.put(model.getResource().getProject(), buildConfigs);
+                if (model.getResource() != null) {
+                	projectsAndBuildConfigs.put(model.getResource().getProject(), Collections.emptyList());
+                }
```

Comment:
this looks as if we wont store no build configs at all (if the model holds a resource, we store an empt list of buildconfigs for the project of that resource). We also dont get the build configs any more. Without testing it, this doesnt look correct. Is it?